app/src/main/java/com/duckduckgo/app/browser/localstorage/LocalStorageManager.kt

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.localstorage
+
+import android.content.Context
+import com.duckduckgo.app.pixels.remoteconfig.AndroidBrowserConfigFeature
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import com.squareup.anvil.annotations.ContributesTo
+import dagger.Module
+import dagger.Provides
+import dagger.SingleInstanceIn
+import java.io.File
+import java.nio.charset.StandardCharsets
+import javax.inject.Inject
+import javax.inject.Provider
+import kotlinx.coroutines.runBlocking
+import org.iq80.leveldb.DB
+import org.iq80.leveldb.Options
+import org.iq80.leveldb.impl.Iq80DBFactory.factory
+import timber.log.Timber
+
+interface LocalStorageManager {
+    fun clearLocalStorage()
+}
```

Comment:
nit: should we call this `WebLocalStorageManager`. `LocalStorageManager` gives the impression that is generic local storage, while this is confined to web only (?)